hudi-utilities/src/test/java/org/apache/hudi/utilities/config/SourceTestConfig.java

```diff
@@ -21,29 +21,36 @@
 
 import org.apache.hudi.common.config.ConfigProperty;
 
+import static org.apache.hudi.common.util.ConfigUtils.DELTA_STREAMER_CONFIG_PREFIX;
+import static org.apache.hudi.common.util.ConfigUtils.STREAMER_CONFIG_PREFIX;
+
 /**
  * Configurations for Test Data Sources.
  */
 public class SourceTestConfig {
 
   public static final ConfigProperty<Integer> NUM_SOURCE_PARTITIONS_PROP = ConfigProperty
-      .key("hoodie.deltastreamer.source.test.num_partitions")
+      .key(STREAMER_CONFIG_PREFIX + "source.test.num_partitions")
```

Comment:
Could you also check the usages of all the `ConfigProperty` definitions and replace all config fetching using `.key()`, e.g., `props.getInteger(SourceTestConfig.MAX_UNIQUE_RECORDS_PROP.key(), SourceTestConfig.MAX_UNIQUE_RECORDS_PROP.defaultValue());`, with `ConfigUtils` APIs, e.g., `getStringWithAltKeys`, `getLongWithAltKeys`, etc.?

Reply:
Note that `.withAlternatives` is only considered by `ConfigUtils` APIs, not by `props.getInteger` with `.key()`.

Reply:
Didn't know that, thanks for comment. Found such usages in AbstractBaseTestSource and DistributedTestDataSource, fixed.